docs/modular-resources/upload/_index.md

```diff
@@ -55,12 +63,11 @@ viam module create --name <your-module-name> --org-id <your-org-id>
 viam module create --name <your-module-name> --public-namespace <your-unique-namespace>
 ```
 
+This command creates a new `meta.json` metadata file in your current working directory, which serves as a template.
+
 {{% /tab %}}
 {{< /tabs >}}
 
-This command creates a new `meta.json` metadata file in your current working directory, which serves as a template.
-Edit and then upload the `meta.json` file to set important configuration information about your module, such as whether it will be publicly available to all Viam users or only available within your organization.
-
 3. Edit the newly-created `meta.json` file, and provide the required configuration information for your custom module by filling in the following fields.
    The `name` field is pre-populated using the `--name` you provided in the `viam module create` command, and `visibility` is set to `private` by default.
 
```

Comment:
[opt] Some people have modified their `module_id` field without understanding that it is what uniquely identifies their module (and then they file confusing bug reports about missing modules when they try to update their module).

Might be worth it to write a short comment saying that module id should not be changed. Up to you